<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Gui::Pref::TInterface</class>
 <widget class="QWidget" name="Gui::Pref::TInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>620</height>
   </rect>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="interface_tab">
      <attribute name="title">
       <string>&amp;Interface</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <property name="spacing">
        <number>16</number>
       </property>
       <property name="margin">
        <number>16</number>
       </property>
       <item>
        <widget class="QGroupBox" name="interface_group">
         <property name="title">
          <string>Interface</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="textLabel1_18">
              <property name="text">
               <string>&amp;Language:</string>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
              <property name="buddy">
               <cstring>language_combo</cstring>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="style_label">
              <property name="text">
               <string>&amp;Style:</string>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
              <property name="buddy">
               <cstring>style_combo</cstring>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="Gui::Pref::TComboBox" name="style_combo">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="Gui::Pref::TComboBox" name="language_combo">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <spacer>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="1" column="1">
             <widget class="Gui::Pref::TComboBox" name="iconset_combo">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="iconset_label">
              <property name="text">
               <string>&amp;Icon set:</string>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
              <property name="buddy">
               <cstring>iconset_combo</cstring>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="main_window_group">
         <property name="title">
          <string>Main window</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <layout class="QHBoxLayout">
          <property name="spacing">
           <number>16</number>
          </property>
          <item>
           <widget class="QLabel" name="instances_icon">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout">
            <property name="spacing">
             <number>6</number>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
            <item>
             <widget class="QCheckBox" name="single_instance_check">
              <property name="text">
               <string>&amp;Use only one window running WZPlayer</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="save_size_check">
              <property name="text">
               <string>&amp;Remember position and size</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="resize_on_load_check">
              <property name="text">
               <string>R&amp;esize window when loading a new video</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="hide_video_window_on_audio_check">
              <property name="text">
               <string>Hide &amp;video window when playing audio files</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="pause_on_minimize_check">
              <property name="text">
               <string>Pause when &amp;minimized</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="close_on_finish_check">
              <property name="text">
               <string>Cl&amp;ose when playback finished</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="fullscreen_toolbars_group">
         <property name="title">
          <string>Fullscreen</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="hide_toolbars_label">
              <property name="text">
               <string>Hide tool&amp;bars after</string>
              </property>
              <property name="buddy">
               <cstring>hide_toolbars_spin</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="hide_toolbars_spin">
              <property name="suffix">
               <string/>
              </property>
              <property name="minimum">
               <number>500</number>
              </property>
              <property name="maximum">
               <number>36000000</number>
              </property>
              <property name="singleStep">
               <number>500</number>
              </property>
              <property name="value">
               <number>3000</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="ms_label">
              <property name="text">
               <string>milliseconds</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QCheckBox" name="start_fullscreen_check">
            <property name="text">
             <string>Start videos in &amp;fullscreen</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="history_tab">
      <attribute name="title">
       <string>&amp;History</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>16</number>
       </property>
       <property name="leftMargin">
        <number>16</number>
       </property>
       <property name="topMargin">
        <number>16</number>
       </property>
       <property name="rightMargin">
        <number>16</number>
       </property>
       <item>
        <widget class="QGroupBox" name="history_group">
         <property name="title">
          <string>History</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <property name="bottomMargin">
           <number>4</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="max_files_label">
              <property name="text">
               <string>Number of recent &amp;files:</string>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
              <property name="buddy">
               <cstring>recents_max_items_spin</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="recents_max_items_spin">
              <property name="minimum">
               <number>0</number>
              </property>
              <property name="maximum">
               <number>100</number>
              </property>
             </widget>
            </item>
            <item>
             <spacer>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="textLabel1_16">
              <property name="text">
               <string>Number of recent &amp;URLs</string>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
              <property name="buddy">
               <cstring>url_max_items_spin</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="url_max_items_spin">
              <property name="minimum">
               <number>0</number>
              </property>
              <property name="maximum">
               <number>50</number>
              </property>
             </widget>
            </item>
            <item>
             <spacer>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QCheckBox" name="save_dirs_check">
            <property name="text">
             <string>&amp;Remember last directory</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Gui::Pref::TComboBox</class>
   <extends>QComboBox</extends>
   <header>gui/pref/combobox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>language_combo</tabstop>
  <tabstop>iconset_combo</tabstop>
  <tabstop>style_combo</tabstop>
  <tabstop>single_instance_check</tabstop>
  <tabstop>save_size_check</tabstop>
  <tabstop>resize_on_load_check</tabstop>
  <tabstop>hide_video_window_on_audio_check</tabstop>
  <tabstop>pause_on_minimize_check</tabstop>
  <tabstop>close_on_finish_check</tabstop>
  <tabstop>hide_toolbars_spin</tabstop>
  <tabstop>start_fullscreen_check</tabstop>
  <tabstop>recents_max_items_spin</tabstop>
  <tabstop>url_max_items_spin</tabstop>
  <tabstop>save_dirs_check</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
